src/handlebars/releases.handlebars

```diff
@@ -82,12 +82,18 @@
                             <div class='small-dl-text'>\{{installer_extension}}</div>
                           </div>
                         </a>
+                        <a href='\{{link}}' class='latest-download-button latest-download-button-grey a-button'>
+                          <div class='large-dl-text'>Download \{{type}}
+                            <div class='small-dl-text'>\{{extension}} - \{{size}} MB</div>
```

Comment:
I'm -0 to showing the extension.  Not sure how many people will care to know that ahead of downloading, but don't feel strongly about it.

Reply:
removed